store/mockstore/unistore/cophandler/mpp_exec.go

```diff
@@ -0,0 +1,379 @@
+package cophandler
+
+import (
+	"context"
+	"io"
+	"math"
+	"sync"
+	"time"
+
+	"github.com/ngaut/unistore/tikv/dbreader"
+	"github.com/pingcap/errors"
+	"github.com/pingcap/kvproto/pkg/kvrpcpb"
+	"github.com/pingcap/kvproto/pkg/mpp"
+	"github.com/pingcap/tidb/expression"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/sessionctx/stmtctx"
+	"github.com/pingcap/tidb/store/tikv/tikvrpc"
+	"github.com/pingcap/tidb/tablecodec"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/chunk"
+	"github.com/pingcap/tidb/util/codec"
+	"github.com/pingcap/tidb/util/rowcodec"
+	"github.com/pingcap/tipb/go-tipb"
+)
+
+// mpp executor that only servers for mpp execution
+type mppExec interface {
+	open() error
+	next() (*chunk.Chunk, error)
+	getFieldTypes() []*types.FieldType
```

Comment:
why not have close()?

Reply:
"close()" is for release resources on time. It's pointless here.